src/server.rs

```diff
@@ -392,7 +426,11 @@ mod test {
     }
 
     fn device_id() -> String {
-        thread_rng().gen_ascii_chars().take(8).collect()
+        iter::repeat(())
```

Comment:
there's a sample_iter for this: `thread_rng().sample_iter(&distributions::Alphanumeric).take(8).collect()`

Reply:
This is handled in a later pr: https://github.com/mozilla-services/rustbox/pull/9/files#diff-4ce93534efc34e923ce01e975eb7ed80R432